assets/src/components/kubernetes/rbac/RBAC.tsx

```diff
@@ -38,7 +38,7 @@ const directory = [
   { path: SERVICE_ACCOUNTS_REL_PATH, label: 'Service Accounts' },
 ] as const
 
-export default function Access() {
+export default function RBAC() {
```

Comment:
I'm not sure the convension around casing these acronyms, but i feel like `Rbac` and `Rbac.tsx` seems more correct perhaps?

Reply:
fixed

Reply:
I think the filename is still `RBAC.tsx` though

Reply:
Yeah just realized git ignored the casing change. fixed